components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -193,23 +196,33 @@ export class ProjectEffects {
   @Effect()
   getApplyRulesStatus$ = this.actions$.pipe(
     ofType<GetApplyRulesStatus>(ProjectActionTypes.GET_APPLY_RULES_STATUS),
-    mergeMap(() =>
-      this.requests.getApplyRulesStatus().pipe(
-        map((resp) => new GetApplyRulesStatusSuccess(resp)),
-        catchError((error: HttpErrorResponse) =>
-          observableOf(new GetApplyRulesStatusFailure(error))))));
-
-  @Effect()
-  getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS).pipe(
-    withLatestFrom(this.store.select(iamMajorVersion).pipe(filter(identity))),
-    withLatestFrom(this.store.select(iamMinorVersion).pipe(filter(identity))),
-    filter(([[_, major], minor]) => {
-      return major === 'v2' && minor === 'v1' && major < minor;
-    }),
-    switchMap(() =>
-      this.requests.getApplyRulesStatus().pipe(
-        map((resp) => new GetApplyRulesStatusSuccess(resp)),
-        catchError((error: HttpErrorResponse) =>
-          observableOf(new GetApplyRulesStatusFailure(error))))));
-}
+    switchMap(this.getRulesStatus$()));
 
+  @Effect()
+  getActiveApplyRulesStatus$ = observableInterval(1000 * ACTIVE_RULE_STATUS_INTERVAL).pipe(
```

Comment:
That's true, but the bigger complication is introducing a variable interval. It can be done, but it is more complicated code--this is so simple comparatively, which is why I went with two emitters.